src/controllers/programs-controller.js

```diff
@@ -125,13 +126,18 @@ router.get("/:id", async (req, res) => {
 router.post("/", async (req, res) => {
     try {
         console.log(req.body)
-        const { name, programCode, universityName, programDescription, prerequisite } = req.body;
-
-        if (!name || !programCode || !universityName || !prerequisite) {
+        const name = req.body.programName;
+        const programCode = req.body.programCode;
+        const universityName = req.body.programUniversity;
+        const programDescription = req.body.programDescription;
+        const prerequisite = req.body.programPrerequisites;
+        const tuitionFee = req.body.programTuition;
+
+        if (!name || !programCode || !universityName || !prerequisite || !tuitionFee) {
             return res.status(400).json({ error: "To create a new program, name, program code, university name, and prerequisites are required." });
         }
 
-        const invalidPrerequisites = prerequisite.filter(p => !["Math3B", "Math4", "Math5", "Physics1A", "Physics2", "Chemistry1", "Chemistry2", "Biology1", "Biology2"].includes(p));
+        const invalidPrerequisites = prerequisite.filter(p => !["Math3B", "Math4", "Math5", "Physics1A", "Physics2", "Chemistry1", "Chemistry2", "Biology1", "Biology2", "Science2", "Civics1B", "History1B", "SpecialRequirement"].includes(p));
```

Comment:
### Possible Future Issue
SpecialRequirement seems to be kind of vague.
It can vary based on the program that a user wants to study (ex. music exam).

Reply:
There will be a future requirement to deal with this issue